docker/docker-compose.yml

```diff
@@ -62,6 +65,10 @@ services:
     build:
       context: ..
       dockerfile: ./docker/bigquery-writer/Dockerfile
+    volumes:
+      - ../credentials:/code/credentials
```

Comment:
We should make a decision here if you want the source code of the bigquery writer mounted into the container as a volume too (for easier code changes). This volume will have to be configured for production (`docker run -v .....`), but if we have a volume `..:/code` then this isn't strictly necessary) - to be continued in our discussion on metabrainz docker style guide